config.toml

```diff
@@ -45,17 +45,24 @@ time_format_blog = "Monday, January 02, 2006"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.10"
+latest = "v1.11"
 
-fullversion = "v1.10.3"
-version = "v1.10"
+fullversion = "v1.11.0"
+version = "v1.11"
 githubbranch = "master"
-docsbranch = "master"
+docsbranch = "release-1.11"
 deprecated = false
```

Comment:
@chenopis @steveperry-53 Is this the value we change to add a "static snapshot" warning to 1.10?

Reply:
Yes, it triggers display of the deprecation warning message at the top of the page defined in `/layouts/partials/deprecation-warning.html`.

Reply:
This should be done in the `release-1.10` branch after it has been synced w/ `master` before the 1.11 release.